HW_4/task_26.py

```diff
@@ -0,0 +1,11 @@
+base = int(input("Введите число: "))
+
+exp = int(input("Введите степень: "))
+
+def power(base, exp):
+    if (exp == 1):
+        return (base)
+    if (exp != 1):
+        return (base * power(base, exp - 1))
```

Comment:
Это условие не нужно, так как предыдущее исключит все? для чего это условие дало бы False.
```suggestion

    return base * power(base, exp - 1)
```